.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -13,36 +13,9 @@ jobs:
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
   timeoutInMinutes: 360
+  variables: {}
 
   steps:
-  - script: |
-         sudo mkdir -p /opt/empty_dir || true
-         for d in \
-                  /opt/ghc \
-                  /opt/hostedtoolcache \
-                  /usr/lib/jvm \
-                  /usr/local/.ghcup \
-                  /usr/local/lib/android \
-                  /usr/local/share/powershell \
-                  /usr/share/dotnet \
-                  /usr/share/swift \
-                  ; do
-           sudo rsync --stats -a --delete /opt/empty_dir/ $d || true
-         done
-         sudo apt-get purge -y -f firefox \
-                                  google-chrome-stable \
-                                  microsoft-edge-stable
-         sudo apt-get autoremove -y >& /dev/null
-         sudo apt-get autoclean -y >& /dev/null
-         df -h
-    displayName: Manage disk space
-  # This creates swap file that creates temporary memory space when the system runs low on memory
-  - script: |
-      sudo fallocate -l 10GiB /swapfile || true
```

Comment:
re rendering is also removing the workaround that was added for low memory

Reply:
adding back in